components/Events/Filters.js

```diff
@@ -0,0 +1,236 @@
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState, useReducer, React } from 'react';
+import events from '../../data/event';
```

Comment:
nit: use `allEvents` instead to avoid confusion with using `events` in other places
```suggestion
import allEvents from '../../data/event';
```